src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ILspEditorFeatureDetector.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.VisualStudio.Razor;
 
 internal interface ILspEditorFeatureDetector
 {
-    bool IsLspEditorEnabled();
+    bool IsLspEditorAvailable(string? documentMoniker);
```

Comment:
This doesn't seem to need to be nullable
```suggestion
    bool IsLspEditorAvailable(string documentMoniker);
```

Reply:
I think you are right, removed it